testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/async/AsyncServletTestCase.java

```diff
@@ -76,7 +76,7 @@
         war.addPackage(AsyncServlet.class.getPackage());
         // Take web.xml from the managed test.
         war.setWebXML(SimpleServlet.class.getPackage(), "web.xml");
-        war.addAsWebInfResource(DistributableTestCase.class.getPackage(), "jboss-web_fine.xml", "jboss-web.xml");
+        war.addAsWebInfResource(DistributableTestCase.class.getPackage(), "jboss-all_fine.xml", "jboss-all.xml");
```

Comment:
This test uses the same jboss-all_fine.xml file as FineWebFailoverTestCase.java, what about using a new one for covering the reference to an existing server profile from a jboss-all.xml file? (at present we have it from distributable-web.xml file in testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/persistence/distributable-web.xml)

Reply:
OK - I can do that.  That said, I'm not overly concerned with which configuration file contains the config, as the namespace has only 1 parser, but this is trivial enough to cover.

Reply:
Done.